SetupDataPkg/Docs/ConfigurationFiles/ConfigurationFiles.md

```diff
@@ -32,122 +31,185 @@ This document is intended to describe the Project MU version of Configuration Fi
+[script](../../Tools/WriteConfVarListToUefiVars.py) or to apply via dmpstore in an EFI shell. These have a .vl suffix
+to indicate they are in variable list format.
+
+- Save Full Config Data to Binary:
+  Create a binary with all config knobs included in it.
+- Save Config Changes to Binary:
+  Create a binary with only changed config knobs (as compared to the base XML(s)) included in it.
+- Load Config Data from Binary:
+  Load a saved variable list binary into the UI. This binary can have been generated by ConfigEditor or dmpstore. In
+  the case that any settings are present that are not in the base XML, they will be ignored. In this way, dmpstore can
+  easily dump all UEFI variables and the user can load them in the tool, only seeing the config they care about.
+
+### SVD Operations
 
 The SVD is intended for use with the UEFI [Conf App](../../ConfApp/), which can take the SVD as input
-and give an SVD describing the current UEFI settings as an output. The SVD is formatted to be compatible
-with [DFCI](https://github.com/microsoft/mu_plus/tree/release/202202/DfciPkg).
+and give an SVD describing the current UEFI settings as an output. ConfApp will accept an SVD via USB or serial. After
+receiving the SVD, ConfApp will reboot the system to allow the settings to be applied. The SVD is an XML with base64
+encoded data to be able to pass via serial.
 
 - Save Full Config Data to SVD File:
-    Saving the entire defined YAML/XML structure into encoded binary settings format. This format is useful when many
-    tags of settings need updating at once, but this will save *all* configurations defined.
+    Save all config knobs into SVD format.
 - Save Config Changes to SVD File:
-    Saving only the changed tag setting into corresponding encoded binary value. This will allow the target system to
-    update only the changed tag setting (i.e. Only disable GFX controllers, and leave USB ports on the same system
-    intact)
+    Save only the config knobs that have been changed from the base XML(s) into SVD format. This option creates a
+    smaller SVD and only updates settings that need to be changed.
 - Load Config Data from SVD File:
     Once the target system has dumped current configuration from ConfApp, the output data can be viewed in
     the ConfigEditor on a host system or saved SVDs from the ConfigEditor can be loaded again.
 
-### Change Files (Delta or CSV)
+### Change File Operations
+
+Profiles are represented as delta files in CSV format on top of the generic profile (for more info see the
+[Profiles](../Profiles/Overview.md) doc).
 
-Profiles are represented as delta files on top of the generic profile (for more info see the
-[Profiles](../Profiles/Overview.md) doc). In addition, the XML differences between what is set in the UI and the base
-XML can be saved as CSV files.
+Delta (or change) files may be loaded into the ConfigEditor and it will update the relevant setting with the same

```

Comment:
Yeah, this is an issue I have that is left over from the YAML + XML days where we called them change files (I dislike the name, seems confusing) because they were different file formats between the two. Now there is only XML, so I'd like to call them a better name, but don't want to make another change to the UI tool. I'll just clean up the doc and call them change files to make it less confusing.